src/planner/builder/orderby_plan.go

```diff
@@ -41,32 +40,28 @@ type OrderBy struct {
 
 // OrderByPlan represents order-by plan.
 type OrderByPlan struct {
-	log      *xlog.Log
-	node     sqlparser.OrderBy
-	tuples   []selectTuple
-	tbInfos  map[string]*tableInfo
-	OrderBys []OrderBy `json:"OrderBy(s)"`
-	typ      ChildType
+	log  *xlog.Log
+	node sqlparser.OrderBy
+	root PlanNode
+	// The indexes mark the fields to be removed.
```

Comment:
the comment is about the reson behind, Easy to understand the meaning.
it is not verbose description.